internal/pkg/cli/task_run.go

```diff
@@ -152,6 +158,8 @@ func newTaskRunOpts(vars runTaskVars) (*runTaskOpts, error) {
 		if err != nil {
 			return err
 		}
+
+		opts.configureEventsWriter = opts.eventsWriterConfigurer(sess)
```

Comment:
Maybe we could simplify by
```suggestion
		opts.configureEventsWriter = func(task []*task.Task) {
				logGroupName := fmt.Sprintf(fmtTaskLogGroupName, o.groupName)
		                o.eventsWriter = &task.EventsWriter{
			              GroupName: logGroupName,
			              Tasks: tasks,

			              Describer: ecs.New(sess),
			              EventsLogger: cloudwatchlogs.New(sess),
			              Writer: log.OutputWriter,
			        }
		}
```